source/sass/mozfest.scss

```diff
@@ -39,23 +39,11 @@ body.mozfest {
         }
       }
 
-      .full-width-cutout {
-        $grid-gutter-width: map-get($grid-gutter-widths, xs);
-        $padding-x: #{$grid-gutter-width / 2};
-
+      .cutout {
+        z-index: $banner-background-z-index + 2;
```

Comment:
It seems like we only need this z-index starting at medium breakpoint. What do you think @mmmavis or it doesn't really matter?

Reply:
Ummm I think we can keep it there. If we do this just for medium+ then we'll have to adjust a bunch of other elements' `z-index` for different screen sizes which might make the code less readable.

Reply:
cool cool 👍